llvm.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+################################################################################
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+################################################################################
+#
+# This script will install the llvm toolchain on the different
+# Debian and Ubuntu versions
+
+set -eux
+
+CURRENT_LLVM_STABLE=14
```

Comment:
You shouldn't push this file. This is just for formatting your code the right way. Best to keep it outside the repo.